cost/google/idle_compute_instances/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v2.11
+
+Optimized the execution time of the policy by using the following approaches:
```

Comment:
Each item in the changelog should start with a - to make it part of a list. I would also recommend streamlining the language a bit; the changelog does not have to provide detailed justifications or explanations for changes.

Here is an example that would work:

- CPU and memory utilization data is now retrieved using GCP Monitoring Query Language
- Instance data is now retrieved in bulk requests per zone